### ISSUE TITLE
Support for evaluating expressions

### ISSUE THREAD

**Issue by dlsniper** [derekparker/delve#251]
One of the features of a debugger is to provide support for evaluating expressions.
One example of usage for this is when you have a watch sent on an expression such as:
```
watch a == true
```

```go
package main

func main() {
	a := false
	_ = a // a breakpoint here should evaluate the watch to false
}
```

Setting a breakpoint on line 5 should yield `false` for the evaluation of `a == true`.

I'm pretty sure this is a complex one to do but I haven't done the research yet.

**Comment by ramigg** [comment]
+1

**Comment by gshilin** [comment]
+2
Totally agree!
Please, please implement it!!!

**Comment by aarzilli** [comment]
Expression evaluation was merged a while ago, we have issues #119 and #59 tracking missing features. I'm going to close this, if there is some feature that's needed a new issue specific to that should be opened.
